src/onediff/quantization/quant_pipeline_test.py

```diff
@@ -0,0 +1,42 @@
+from diffusers import AutoPipelineForText2Image
+from onediff.quantization.quantize_pipeline import QuantPipeline
+import torch
+import argparse 
+
+floatting_model_path = "/share_nfs/hf_models/stable-diffusion-xl-base-1.0"
```

Comment:
```suggestion
floatting_model_path = "stabilityai/stable-diffusion-xl-base-1.0"
```
原始 量化文档： https://github.com/siliconflow/onediff/blob/main/README_ENTERPRISE.md#accessing-diffusers-models